### ISSUE TITLE
[Request] Allow bind_rows to take rds/feather file names

### ISSUE THREAD

**Issue by kendonB** [hadley/dplyr#2327]
Related to: https://github.com/wesm/feather/issues/273

This would only allocate the memory for the data only once, rather than twice under what (to my knowledge) is currently possible.

**Comment by hadley** [comment]
This is out of scope for dplyr.
